### PR TITLE
Add TableView DnD docs

### PR DESCRIPTION
**Pull request by reidbarber**
## ✅ Pull Request Checklist:

- [ ] Included link to corresponding [React Spectrum GitHub Issue](https://github.com/adobe/react-spectrum/issues).
- [ ] Added/updated unit tests and storybook for this change (for new code or code which already has tests).
- [ ] Filled out test instructions.
- [ ] Updated documentation (if it already exists for this component).
- [ ] Looked at the Accessibility Practices for this feature - [Aria Practices](https://www.w3.org/TR/wai-aria-practices-1.1/)

### CODE REVIEW

**Comment by reidbarber** [comment]
> Dragging seems to take two clicks the first time for some reason.

Investigating, not sure why this is happening.

**Comment by ktabors** [comment]
Should this be updated to include a custom drag preview section?

**Comment by reidbarber** [comment]
> Should this be updated to include a custom drag preview section?

@ktabors Good idea, I'll add that since it will be merged after release anyway.

File: packages/@react-spectrum/table/docs/TableView.mdx [hunk 1]
```diff
@@ -729,6 +731,855 @@ function ResizableTable() {
...
+
+### Custom drag previews
+
+By default, the drag preview for a dragged row will include the `textValue` provided to that row.
+If no `textValue` was provided, it will default to the contents of the columns that are row headers, as specified by the `isRowHeader` prop on columns.
+If no row header columns were specified, the first column will be used, as mentioned in the [Accessibility](#accessibility) section.
+
+Use the `renderPreview` prop to provide a custom drag preview. `keys` and `draggedKey` are passed to this function, where `keys` includes all the keys of the items being dragged, and `draggedKey` is the key of the item the user initiated the drag from.
+
+```tsx example
```
Comment (ktabors) [inline packages/@react-spectrum/table/docs/TableView.mdx:1517]: Should this be hidden with a "show code" too?
```suggestion
```tsx example export=true render=false
```

**Review by ktabors — DISMISSED** [review]

**Review by dannify — APPROVED** [review]

**Pull request merged**